Documentation/trying-out-rkt.md

```diff
@@ -1,13 +1,14 @@
-# Trying out rkt 
+# Trying out rkt
 
-This guide provides a short introduction to trying out rkt.
-For a more in-depth guide of a full end-to-end workflow of building an application and running it using rkt, check out the [getting-started-guide](getting-started-guide.md)
+This document introduces the basics of trying out rkt. For a more in-depth guide to building an application container and running it with rkt, check out the [getting started guide][getstart].
 
 ## Using rkt on Linux
 
-rkt consists of a single CLI tool and can run on different platforms. The primary target platform currently is amd64 Linux. A modern kernel is required but there should be no other system dependencies.
+rkt consists of a single command line program compiled for Linux on the amd64 architecture. A modern Linux kernel is required, but there are no other system prerequisites.
```

Comment:
It is on purpose -- We do not at this release (1.9) or time deliver any arch but amd64.

When https://github.com/coreos/rkt/pull/2758 merges and there is support, that's when I think this doc gets an update. Until then it just tells the unalloyed fact, because we're specifically documenting the binary rkt here, not instructing on how you might compile for archs that will later be supported.

Consider what we might say instead: "rkt is compiled as an amd64 binary, but could be compiled on other architectures"? -- which ones, how, and does the result work? 

Which brings me to my question about this, that maybe changes my opinion on how we write this sentence: What does debians rkt 1.5 package actually *do* on a ppc64? Does it work?